image/upload/pack.sh

```diff
@@ -38,7 +38,10 @@ pack() {
 
   azure)
     echo "📥 Packing Azure image..."
-    truncate -s %1MiB "${unpacked_image_dir}/${unpacked_image_filename}"
+    # Disk Images on Azure have to be a multiple of 1MiB in size.
+    # Marketplace images additionally have to be at least 1GiB in size.
+    truncate -s %2G "${unpacked_image_dir}/${unpacked_image_filename}" # at least 1GiB
+    # truncate -s +1M "${unpacked_image_dir}/${unpacked_image_filename}" # ensure > 1 GiB, but multiple of 1MiB
```

Comment:
nit: let's document clearly why we chose 2GiB here:

Marketplace validation says >= 1 GiB, but using an image of size 1 GiB actually fails. Otherwise this looks  very confusing.

Reply:
Yes, this is only for temporary testing, to ensure the hour-long marketplace pipeline won't fail. I'll test the commented-out approach (rounding up to a multiple of 1GiB and then adding 1MiB to ensure to be ">") afterwards.